src/main/java/com/untamedears/realisticbiomes/commands/Menu.java

```diff
@@ -22,13 +24,41 @@ public boolean execute(CommandSender sender, String[] args) {
 			p.sendMessage(ChatColor.RED + "You can't use this command in vehicles");
 			return true;
 		}
-		RealisticBiomesGUI gui = new RealisticBiomesGUI((Player) sender);
-		gui.showRBOverview();
+		if (args.length == 0 || args[0].length() == 0) {
+			RealisticBiomesGUI gui = new RealisticBiomesGUI((Player) sender);
+			gui.showRBOverview(null);
+		} else {
+			if (!p.isOp()) {
+				p.sendMessage(ChatColor.RED + "You lack permission to use this command with arguments");
+				return true;
+			}
+			String concat = String.join(" ", args);
+			for (Biome b : Biome.values()) {
+				if (b.toString().equals(concat)) {
+					RealisticBiomesGUI gui = new RealisticBiomesGUI((Player) sender);
+					gui.showRBOverview(b);
+					return true;
+				}
+			}
+			p.sendMessage(ChatColor.RED + "The biome " + concat + " does not exist");
+		}
 		return true;
 	}
 
 	@Override
-	public List<String> tabComplete(CommandSender arg0, String[] args) {
+	public List<String> tabComplete(CommandSender sender, String[] args) {
```

Comment:
Note that for use cases like this we have tab completion methods inherited in StandaloneCommand.

Here you could just do something like
`return doTabComplete(args [0], Biome.values(), Biome::name, false)`;